python/location/lbs/activity/audioset/dataprocessing/audio_processing.py

```diff
@@ -0,0 +1,262 @@
+# This Python file is an extended/modified version of Jerry Wu's audio_prep.py
+# This python file is a script to download a given label from AudioSet
+# The downloaded audios would be clipped into 10 seconds periods
+# The downloaded audio segments are then processed using Librosa to extract features
+# Stored in eval_segments, balanced_train, and unbalanced_train separately.
+
+# Usage: python3 audio_prep.py <label>
+
+
+from __future__ import unicode_literals
+import youtube_dl
+import csv
+import json
+from pydub import AudioSegment
+import os
+import shutil
+from os import listdir
+from os.path import isfile, join
+import sys
+import numpy as np
+import librosa
+import argparse
+import numpy as np
+
+file_structure = "python/location/lbs/activity/audioset/dataprocessing/"
+referDoc = "ReferDoc/"
+downloaded_audio = []
+feature_dict = {}
+
+
+def setup_commandline_args():
+  parser = argparse.ArgumentParser(description='Process dataset.csv to input into TensorFlow.')
+  parser.add_argument('-d', '--datasets', required=True, type=str, nargs='+')
+  parser.add_argument('-r', '--redo', action='store_true', required=False)
+  parser.add_argument('-l', '--labels', required=True, type=str, nargs='+')
+  args = parser.parse_args()
+  print(args)
+  return args
+
+
+def parse_csv(csv_file):
+  # Extract all labeled audio info from csv
```

Comment:
Add a function docstring as described here:

def FetchBigtableRows(big_table, keys, other_silly_variable=None):
  """Fetches rows from a Bigtable.

  Retrieves rows pertaining to the given keys from the Table instance
  represented by big_table.  Silly things may happen if
  other_silly_variable is not None.

  Args:
    big_table: An open Bigtable Table instance.
    keys: A sequence of strings representing the key of each table row
      to fetch.
    other_silly_variable: Another optional variable, that has a much
      longer name than the other args, and which does nothing.

  Returns:
    A dict mapping keys to the corresponding table row data
    fetched. Each row is represented as a tuple of strings. For
    example:

    {'Serak': ('Rigel VII', 'Preparer'),
     'Zim': ('Irk', 'Invader'),
     'Lrrr': ('Omicron Persei 8', 'Emperor')}

    If a key from the keys argument is missing from the dictionary,
    then that row was not found in the table.

  Raises:
    IOError: An error occurred accessing the bigtable.Table object.
  """